File: php/testcode/benchmark_test_00983.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00983(BenchmarkRequest $req): BenchmarkResponse {
    $input = array_merge($_GET, $_POST);
    $model = new stdClass();
    foreach ($input as $k => $v) {
        $model->$k = $v;
    }
    return BenchmarkResponse::ok('updated');
}

File: rust/testcode/benchmark_test_00983.rs
fn db_get_order(order_id: &str) -> String {
    format!("order_details_for_{}", order_id)
}

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let order_id = req.param("order_id");
    let order = db_get_order(&order_id);
    super::shared::BenchmarkResponse::ok(&order)
}
